mmocr/utils/ocr.py

```diff
@@ -12,13 +12,16 @@
 from mmcv.image.misc import tensor2imgs
 from mmcv.runner import load_checkpoint
 from mmcv.utils.config import Config
+from PIL import Image
+from tesserocr import PyTessBaseAPI, RIL
```

Comment:
```suggestion
try:
    import tesserocr
    from tesserocr import PyTessBaseAPI, RIL
except ImportError:
    tesserocr = None
```
`tesserocr` is better to be an optional library for MMOCR